### ISSUE TITLE
Plot GP dist test is flaky

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc#6656]
https://github.com/pymc-devs/pymc/actions/runs/4587854474/jobs/8190466483?pr=6635

**Comment by michaelosthege** [comment]
That's not special to `plot_gp_dist` - it fails in the line `pyplot.subplots()` (!)

https://github.com/pymc-devs/pymc/actions/runs/4587854474/jobs/8190466483?pr=6635#step:7:188

**Comment by ricardoV94** [comment]
Any idea why (and only sometimes)?

**Comment by michaelosthege** [comment]
No, I never saw it before, and I'm no expert in tkinter/matplotlib backends..

**Comment by QuLogic** [comment]
Since this is CI, I assume you don't actually need the interactivity. It would be best to set the backend to a non-interactive one, by setting the `MPLBACKEND=Agg` environment variable. Then you won't have to worry about Tk being broken.

**Comment by ricardoV94** [comment]
Sounds good @QuLogic, do you want to take a stab at it yourself?
